portallocator/portallocator_windows.go

```diff
@@ -1,10 +1,10 @@
 package portallocator
 
-const (
-	StartPortRange = 60000
-	EndPortRange   = 65000
-)
+func init() {
```

Comment:
same question. Is there any specific reason for  moving away from const ?

Reply:
ditto